client/env

```diff
@@ -0,0 +1,7 @@
+VITE_FIREBASE_API_KEY="AIzaSyCu7Z75EZjWobqG4wtRZ_Bxjrc6MjHUMhE"
```

Comment:
This file should not be on github!
Please rename this file to `.env` and ensure it is not being tracked by git.